pkg/core/blockchain_neotest_test.go

```diff
@@ -70,6 +70,7 @@ func TestBlockchain_StartFromExistingDB(t *testing.T) {
 	customConfig := func(c *config.Blockchain) {
 		c.StateRootInHeader = true // Need for P2PStateExchangeExtensions check.
 		c.P2PSigExtensions = true  // Need for basic chain initializer.
+		c.Hardforks = nil
```

Comment:
This change here is also suspicious. Revert it and fix failing test cases instead. If you set hardforks to nil, then all hardforks are enabled from genesis by default, and this testing scenario is the easiest one, whereas our goal is to ensure that native update works properly.